nav2_dijkstra_planner/src/dijkstra_planner.cpp

```diff
@@ -85,7 +85,13 @@ DijkstraPlanner::execute(const nav2_tasks::ComputePathToPoseCommand::SharedPtr c
       costmap_.metadata.size_x, costmap_.metadata.size_y);
 
     // Create a planner based on the new costmap size
-    planner_ = std::make_unique<NavFn>(costmap_.metadata.size_x, costmap_.metadata.size_y);
+    if (!planner_.get() || current_costmap_size_[0] != costmap_.metadata.size_x ||
```

Comment:
It took me a few moments to understand what this comparison was doing. It would be nice to make a function like
```c++
bool isPlannerOutOfDate()
{
  return !planner_.get() || 
    current_costmap_size_[0] != costmap_.metadata.size_x || 
    current_costmap_size_[1] != costmap_.metadata.size_y;
}

Reply:
done